crates/iroha_genesis/Cargo.toml

```diff
@@ -13,7 +13,7 @@ workspace = true
 [dependencies]
 iroha_crypto = { workspace = true }
 iroha_schema = { workspace = true }
-iroha_data_model = { workspace = true, features = ["http"] }
+iroha_data_model = { workspace = true, features = ["std"] }
```

Comment:
it doesn't make sense that `http` was required for genesis. I just moved `TransactionBuilder` out of `http` feature in `data_model`